mongo/src/test/java/org/axonframework/extensions/mongo/eventsourcing/eventstore/MongoEventStorageEngineTest.java

```diff
@@ -92,10 +92,8 @@ public void setUp() {
         mongoTemplate.snapshotCollection().dropIndexes();
         mongoTemplate.eventCollection().deleteMany(new BasicDBObject());
         mongoTemplate.snapshotCollection().deleteMany(new BasicDBObject());
-        testSubject = context.getBean(MongoEventStorageEngine.class);
+        testSubject = MongoEventStorageEngine.builder().mongoTemplate(mongoTemplate).build();
```

Comment:
This is the line which bothers me.

Reply:
Thanks for this @mwlynch, I'll merge it for now and try to fix the issue to be able to release 4.1 as desired.